Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/viewmodel/SubmissionQRCodeInfoFragmentViewModel.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.ui.submission.viewmodel
+
+import androidx.lifecycle.ViewModel
+import de.rki.coronawarnapp.util.ui.SingleLiveEvent
+
+class SubmissionQRCodeInfoFragmentViewModel : ViewModel() {
+
+    val navigateBack = SingleLiveEvent<Boolean>()
```

Comment:
The overall navigation pattern looks odd to me for the following reasons:
- The boolean value will always be true (false will never be emitted)
- Sending an event to the VM and observing the same thing in the Fragment is somewhat duplicate. What's the advantage over calling the navigation in the fragment directly?
- If we go down that route, a navigate state/enum BACK and FORWARD and single Event to be observed and handled depending on the type might be more straight forward.
- tests are missing for the VM